pycbc/waveform/generator.py

```diff
@@ -277,10 +293,14 @@ def generate(self, *args):
         rfparams = dict([(param, self.current_params[param])
             for param in self.rframe_generator.variable_args])
         hp, hc = self.rframe_generator.generate_from_kwargs(**rfparams)
+        hp._epoch = hc._epoch = self._epoch
```

Comment:
Yes, that's right Chris. This should make it easier for setting the ini files anyway, since we'll have a GPS time of a waveform.